docs/strings.rst

```diff
@@ -28,6 +28,24 @@ will be treated as variable-length strings, marked as UTF-8 and ASCII respective
 Numpy bytes arrays (``'S'`` dtypes) make fixed-length strings.
 You can use :func:`.string_dtype` to explicitly specify any HDF5 string datatype.
 
+::
+
+    string_data = ["varying", "sizes", "of", "strings"]
+
+    # Variable length strings (implicit)
+    f['vlen_strings1'] = string_data
```

Comment:
This `f` is supposed to be the database? What is supposed to happen when you are doing it implicitly?

Reply:
`f` for file - this is a convention across the h5py docs, e.g. look further down this page, or at the [dataset page](https://docs.h5py.org/en/stable/high/dataset.html). We don't really use the word database at all; HDF5 also talks about files rather than databases.

'Implicit' here just means you're not telling h5py a dtype, so it's guessing based on the object you give it.